extension/scripts/content/global/ttGlobal.entry.js

```diff
@@ -1,70 +1,234 @@
+		featureManager.load("Hide Leave Buttons");
+		featureManager.load("Hide Icons");
+		featureManager.load("Hide Areas");
+		// featureManager.load("Hide Chats");
+		// featureManager.load("Clean Flight");
+		featureManager.load("Chat Font Size");
+		featureManager.load("Highlight Energy Refill");
+		featureManager.load("Highlight Nerve Refill");
+	}
 
-	forceUpdate().catch(() => {});
-}
+	alignLeft() {
+		featureManager.new({
+			name: "Align Left",
+			scope: "global",
+			enabled: settings.pages.global.alignLeft,
+			func: feature,
+			runWhenDisabled: true,
+		});
+
+		async function feature() {
+			if (settings.pages.global.alignLeft) document.documentElement.classList.add("tt-align-left");
+			else document.documentElement.classList.remove("tt-align-left");
+		}
+	}
+	hideLevelUpgrade() {
+		featureManager.new({
+			name: "Hide Level Upgrade",
+			scope: "global",
+			enabled: settings.pages.global.hideLevelUpgrade,
+			func: feature,
+			runWhenDisabled: true,
+		});
+
+		async function feature() {
+			document.documentElement.style.setProperty("--torntools-hide-upgrade-button", settings.pages.global.hideLevelUpgrade ? "none" : "block");
+		}
+	}
+	hideLeaveButton() {
+		featureManager.new({
+			name: "Hide Leave Buttons",
+			scope: "global",
+			enabled: settings.pages.global.hideQuitButtons,
+			func: feature,
+			runWhenDisabled: true,
+		});
+
+		async function feature() {
+			document.documentElement.style.setProperty("--torntools-hide-leave-button", settings.pages.global.hideQuitButtons ? "none" : "flex");
+		}
+	}
+	hideIcons() {
+		featureManager.new({
+			name: "Hide Icons",
+			scope: "global",
+			enabled: settings.hideIcons.length > 0,
+			func: feature,
+			runWhenDisabled: true,
+		});
+
+		async function feature() {
+			for (let icon of ALL_ICONS) {
+				document.documentElement.style.setProperty(`--torntools-hide-icons-${icon}`, settings.hideIcons.includes(icon) ? "none" : "initial");
+			}
+		}
+	}
+	hideAreas() {
+		featureManager.new({
+			name: "Hide Areas",
+			scope: "global",
+			enabled: settings.hideAreas.length > 0,
+			func: feature,
+			runWhenDisabled: true,
+		});
+
+		async function feature() {
+			for (let area of ALL_AREAS.map((area) => area.class)) {
+				document.documentElement.style.setProperty(`--torntools-hide-area-${area}`, settings.hideAreas.includes(area) ? "none" : "initial");
+			}
+		}
+	}
+	hideChats() {
+		featureManager.new({
+			name: "Hide Chats",
+			scope: "global",
+			enabled: settings.hideChats,
+			func: feature,
+			runWhenDisabled: true,
+		});
+
+		async function feature() {
+			// todo
+		}
+	}
+	cleanFlight() {
+		featureManager.new({
+			name: "Clean Flight",
+			scope: "global",
+			enabled: settings.cleanFlight,
+			func: feature,
+			runWhenDisabled: true,
+		});
+
+		async function feature() {
+			// todo
+		}
+	}
+	chatFontSize() {
+		featureManager.new({
+			name: "Chat Font Size",
+			scope: "global",
+			enabled: settings.pages.chat.fontSize,
```

Comment:
I would say that this feature isn't enabled when the fontSize = 12 (Torn default).

Reply:
Changed.